src/main/java/uk/gov/companieshouse/web/accounts/controller/govuk/GovukCriteriaController.java

```diff
@@ -52,13 +52,13 @@ public String postGovUkCriteria(@ModelAttribute("criteria") @Valid Criteria crit
 
         if (criteria.getIsCriteriaMet().equalsIgnoreCase("noAlternativeFilingMethod")) {
 
-            return UrlBasedViewResolver.REDIRECT_URL_PREFIX + "/accounts/alternative-filing-options";
+            return UrlBasedViewResolver.REDIRECT_URL_PREFIX
+                + "/accounts/alternative-filing-options";
         } else if (criteria.getIsCriteriaMet().equalsIgnoreCase("noOtherAccounts")) {
 
             return UrlBasedViewResolver.REDIRECT_URL_PREFIX + "/accounts/select-account-type";
         }
-
-        //TODO - To be remove along with next controller annotation when "What is the company number" screen is added
-        return navigatorService.getNextControllerRedirect(this.getClass());
+        attributes.addAttribute("forward", "/company/{companyNumber}/small-full/steps-to-complete");
+        return UrlBasedViewResolver.REDIRECT_URL_PREFIX + "/company-lookup/search";
```

Comment:
This change breaks the following unit test: `GovukCriteriaControllerTest.postRequestCriteriaMet`

Reply:
fixed